scripts/common

```diff
@@ -10,6 +10,9 @@ set -o pipefail
 # https://docs.rs/env_logger
 export RUST_LOG="${RUST_LOG:-info}"
 
+# Reduce build nondeterminism by remapping paths so that they don't include local segments.
+export RUSTFLAGS="--remap-path-prefix=$PWD=/"
```

Comment:
Might be nice to take into account any existing value of `RUSTFLAGS`  (same concern about stomping on user-specified values also applies to `scripts/run_tests_tsan` btw)

Reply:
Good point, done.